src/internal/operators/repeatWhen.ts

```diff
@@ -35,7 +35,7 @@ import { createOperatorSubscriber } from './OperatorSubscriber';
  *
  * @param {function(notifications: Observable): Observable} notifier - Receives an Observable of notifications with
  * which a user can `complete` or `error`, aborting the repetition.
- * @return A function that returns an Observable that that mirrors the source
+ * @return A function that returns an Observable that mirrors the source
```

Comment:
There are times one that can be doubled. I'm not entirely confident this isn't one of those times, but I hope I'm right.